lib/core/presentation/theming/colors/colors.dart

```diff
@@ -8,10 +8,10 @@ class _AppColors {
   static Color onErrorW = const Color(0xffFF000D);
   // static Color bgColorW = const Color(0xffF7F7F7);
   static Color bgColorW = Color(0xffFFFFFF);
-  static Color onBgColorW = Color(0xffD9D9D9);
+  static Color onBgColorW = Color(0xffF5F5F5);
   // static Color surfaceW = const Color(0xffF5F5F5); // ;
   // static Color surfacerW = Color(0xffFFFFFF);
-  static Color onSurfaceW = Color(0xffD9D9D9); // const Color(0xff808191);
+  static Color onSurfaceW = Color(0xff888888); // const Color(0xff808191);
 
   static Color getShade(Color color, {bool darker = false, double value = .1}) {
```

Comment:
@TikuGaelle please the onSurfaceW color code is `#F7F7F7` if the page your are currently work on and the background color is completely white (different from the onSurfaceW) use this for your scaffold background `backgroundColor: Theme.of(context).cardColor` rather than modifying the onSurfaceW

Reply:
Thanks @DesmondTambe